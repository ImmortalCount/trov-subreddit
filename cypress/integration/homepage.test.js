describe('Home page', () => {
    it('should render the home page', () => {
        cy.visit('/').then(() => {
            cy.get('header').toMatchSnapshot()
            cy.get('nav').toMatchSnapshot()
            cy.get('input')
                .type('pics')
            // cy.get('form').submit().then(() => {
            //     const items = cy.get('li')
            //     expect(items.length).to.eq(26)
            // })
        })
    })

    // it('should display errors', () => {
    //     cy.visit('/')
    //     cy.get('input').first().type('jason.klingstedt@gmail.com')
    //     cy.get('input').last().type('password')
    //     cy.get('form').submit()
    // })

    // it('should log on as admin', () => {
    //     cy.login('admin')
    //     cy.visit('/').then(() => {
    //         cy.get('#root').toMatchSnapshot()
    //     })
    // })
})
