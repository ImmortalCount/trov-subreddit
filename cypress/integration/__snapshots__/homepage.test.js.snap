exports[`Home page > should render the home page #0`] = `
<header class="sc-ifAKCX gcPEBf">
  <a class="sc-bxivhb drpxUl" href="/">Home</a>
  <fieldset class="sc-htpNat PiYSI">
    <label for="subreddit-select">Select a Suggested Subreddit</label
    ><select id="subreddit-select" class="sc-EHOje jvXsky"
      ><optgroup label="Select a Suggested Subreddit"
        ><option value="reactjs">reactjs</option
        ><option value="guitarpedals">Guitar Pedals</option
        ><option value="guitar">Guitar</option
        ><option value="dataisbeautiful">dataisbeautiful</option
        ><option value="DIY">DIY</option
        ><option value="doommetal">Doom Metal</option
        ><option value="javascript">javascript</option
        ><option value="nottheonion">Not The Onion</option></optgroup
      ></select
    >
  </fieldset>
  <form class="sc-bZQynM epBIuB">
    <input name="subreddit" placeholder="Enter a subreddit..." /><button
      color="green"
      class="sc-bwzfXH cEwmJl"
    >
      submit
    </button>
  </form>
</header>
`;

exports[`Home page > should render the home page #1`] = `
<nav class="sc-gzVnrw fgSTpW">
  <h2>Suggested Subreddits</h2>
  <ul class="sc-htoDjs emIcqi">
    <li><a href="/r/reactjs">reactjs</a></li>
    <li><a href="/r/guitarpedals">Guitar Pedals</a></li>
    <li><a href="/r/guitar">Guitar</a></li>
    <li><a href="/r/dataisbeautiful">dataisbeautiful</a></li>
    <li><a href="/r/DIY">DIY</a></li>
    <li><a href="/r/doommetal">Doom Metal</a></li>
    <li><a href="/r/javascript">javascript</a></li>
    <li><a href="/r/nottheonion">Not The Onion</a></li>
  </ul>
</nav>
`;
